cobalt/build/build_info.py

```diff
@@ -77,13 +78,27 @@ def _get_subject_from_last_commit(cwd):
   return _get_last_commit_with_format(r'%s', cwd=cwd)
 
 
+def _get_cobalt_version():
+  version_header_path = os.path.join(os.path.dirname(FILE_DIR), 'version.h')
+  contents = ''
+  with open(version_header_path, 'r', encoding='utf-8') as f:
+    contents = f.read()
+  compiled_cobalt_version_pattern = re.compile(
+      _COBALT_VERSION_PATTERN, flags=re.MULTILINE)
+  match_cobalt_version = compiled_cobalt_version_pattern.search(contents)
+  if not match_cobalt_version:
+    return ''
```

Comment:
Seems to me this should fail if the version string couldn't be extracted. That does not seem like a scenario that we ever expect to happen.

Reply:
ack